hunting-buddy.lic

```diff
@@ -105,9 +105,11 @@ class HuntingBuddy
     wait_for_script_to_complete('buff', ['prehunt_buffs'])
   end
 
-  def find_hunting_room?(zone_name)
+ def find_hunting_room?(zone_name)
     UserVars.friends = @settings.hunting_buddies || []
-    if rooms = @hunting_zones[zone_name]
+    rooms = []
+    rooms = Array(zone_name).each { |name| rooms.push(*@hunting_zones[name]).flatten; } 
+    if rooms
```

Comment:
This check is kind of meaningless now since you set rooms to an array, it should probably be `unless rooms.empty?`.